presto-main/src/main/java/com/facebook/presto/metadata/StaticFunctionNamespaceStore.java

```diff
@@ -78,7 +84,14 @@ private void loadFunctionNamespaceManager(String catalogName, Map<String, String
         String functionNamespaceManagerName = properties.remove(FUNCTION_NAMESPACE_MANAGER_NAME);
         checkState(!isNullOrEmpty(functionNamespaceManagerName), "%s property must be present", FUNCTION_NAMESPACE_MANAGER_NAME);
 
-        functionAndTypeManager.loadFunctionNamespaceManager(functionNamespaceManagerName, catalogName, properties);
+        if (Boolean.parseBoolean(properties.get(DEFAULT_NAMESPACE))) {
```

Comment:
```suggestion
        if (parseBoolean(properties.get(DEFAULT_NAMESPACE))) {
```

Reply:
I think you'll want to remove this from the properties before sending it to the plugin, in the same way we remove the name.  See line 84.